plugins/debugservices/lib_debugservices.ecllib

```diff
@@ -0,0 +1,4 @@
+export DebugServices := SERVICE
```

Comment:
Also missing plugin attribute (and in other places below)

Reply:
Resolved